src/lib.rs

```diff
@@ -400,6 +402,10 @@ fn parse_outpoint(s: &str) -> Result<OutPoint, String> {
     OutPoint::from_str(s).map_err(|e| format!("{:?}", e))
 }
 
+fn parse_network(s: &str) -> Result<Network, String> {
```

Comment:
small doubt, can we avoid this since Network already has `from_str` -> Result<Network, Error>?

Reply:
Good idea, I thought it needed a function that returned Result<Network,String> but just using `Network::from_str` works fine.